www/casestudies/items/endtoend/authoring.php

```diff
@@ -0,0 +1,154 @@
+
+<?php
+
+include_once '../../../config.php';
+include_once 'includes/header.php';
+
+use LearnositySdk\Request\Init;
+use LearnositySdk\Utils\Uuid;
+
+
+$item_ref = Uuid::generate();
+
+$security = array(
+    'consumer_key' => $consumer_key,
+    'domain'       => $domain
+);
+
+$request = array(
+    'mode'      => 'item_edit',
+    'reference' => $item_ref,
+    'config'    => array(
+        'item_edit' => array(
+            'item' => array(
+                'template' => array(
+                    'type' => 'single_question'
+                )
+            ),
+            'widget' => array(
+                'edit'   => true,
+                'delete' => false
+            )
+        ),
+        'question_editor_init_options' => array(
+            'ui' => array(
+                'public_methods'     => array(),
+                'question_tiles'     => false,
+                'documentation_link' => false,
+                'change_button'      => true,
+                'source_button'      => false,
+                'fixed_preview'      => true,
+                'advanced_group'     => false,
+                'search_field'       => false
+            )
+        )
+    ),
+    'user' => array(
+        'id'        => 'demos-site',
+        'firstname' => 'Demos',
+        'lastname'  => 'User',
+        'email'     => 'demos@learnosity.com'
+    )
+);
+
+$Init = new Init('author', $security, $consumer_secret, $request);
+$signedRequest = $Init->generate();
+
+?>
+
+<div class="jumbotron section">       
+    <div class="overview">
+        <h1>End to End Demo – Authoring</h1>
+        <p>Learnosity's Author API enable professional authors (as well as teachers) to create and edit content.</p>
+        <p>Questions are saved to the Learnosity item bank to be included in the assessment.</p>
+    </div>
+</div>
+
+<div class="section">        
+    <div class="row">
+        <div id="notifications" class="col-md-2"></div>
+        <div class="col-md-10">
+            <div id="learnosity-author"></div>
+        </div>
+    </div>  
+    <p class="text-right">
+        <br>
+        <a class="btn btn-primary btn-md" href="javascript:addNextQuestion();">Add more</a>
```

Comment:
I'd remove the javascript from the href link, and instead add it via an on('click') handler in JS.